src/app/store.ts

```diff
@@ -1,10 +1,21 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import `combineSlices` should be `combineReducers` from `@reduxjs/toolkit`. `combineSlices` is not a valid method and will cause an error.